docs/import/add_disaster.js

```diff
@@ -408,14 +407,16 @@ function initializeScoreSelectors(states) {
     // TODO: Make column names selects instead.
     const columnCell = createTd();
     for (let i = 0; i < scoreAssetTypes[scoreAssetType].length; i++) {
+      columnCell.append($(document.createElement('label'))
+                            .text(scoreAssetTypes[scoreAssetType][i]));
```

Comment:
Might be nice to standardize the sizes of these label + input boxes so the boxes line up down the column. But this comment is pending our other discussion of whether or not we want to handle that in this PR.